babybuddy/views.py

```diff
@@ -147,7 +149,16 @@ def post(self, request):
         return render(request, self.template_name, {"form": form})
 
 
-class UserSettings(LoginRequiredMixin, View):
+class RegenerateApiKey:
+    def handle_api_key_post(self, request) -> Boolean:
```

Comment:
Should this be just `bool`? I noticed `Boolean` here is imported from `xmlrpc.client` and that seems odd...

Reply:
Whoops. You are right. Carry-over from my Android Java/Kotlin project, where the types have different names. Auto-import was _too powerful_ and found a type for "Boolean" anyway :laughing:

Reply:
Also noticed that this does not need to be a class-method anymore. Refactored it to be a plain method.